lib/version_utils.pm

```diff
@@ -314,6 +315,16 @@ sub is_alp {
     return check_version($query, $version, qr/\d{1,}\.\d/);
 }
 
+=head2 is_agama
+
+Check if agama installation is being used
+=cut
+
+sub is_agama {
+    return 1 if ((is_sle('>=16') || is_leap('>=16.0')) && (get_var('AGAMA') || get_var('AGAMA_AUTO'));
+    return 0;
```

Comment:
Please don't do such things, this really bad coding style which sneaked into code base recently.  Just use   `return ((get_var('AGAMA') || get_var('AGAMA_AUTO'));` There is no  need for logic like `return 1 if 1`. Also remove distribution checks completely. Agama is/will be installer for any product.